meson.build

```diff
@@ -378,25 +378,51 @@ install_data(pgroonga_database_update_sqls,
   install_dir: pg_sharedir / 'extension',
 )
 
-pg_regress = pg_pkglibdir / 'pgxs/src/test/regress/pg_regress'
-run_command('mkdir', '-p', meson.current_build_dir() / 'results/function/highlight-html/', check: true)
-run_command('mkdir', '-p', meson.current_build_dir() / 'results/function/command/', check: true)
+ruby = find_program('ruby')
+pg_regress = find_program(
+  'pg_regress',
+  dirs: [pg_pkglibdir / 'pgxs/src/test/regress']
+)
 
-test('regression',
-  find_program(pg_regress),
-  args: [
-    '--inputdir=' + meson.current_source_dir(),
-    '--outputdir=' + meson.current_build_dir(),
-    '--bindir=' + pg_bindir,
-    '--dlpath=' + meson.current_build_dir(),
-    '--load-extension=pgroonga',
-    'function/highlight-html/multiple-keywords',
-    'function/command/select'
+schedule = meson.current_build_dir() / 'schedule'
+prepare_test = custom_target(
+  'prepare',
+  output: 'schedule',
```

Comment:
Is this re-generated when `sql/**.sql` are changed?

Reply:
Yes it is.

Reply:
fix: [b46fd1d](https://github.com/pgroonga/pgroonga/pull/753/commits/b46fd1d348b746647f8d4c6ced90fd9b6d23d63b) I misunderstood your comment. I confirmed that the changes were reflected in *.sql file when we modified the content of it.
Now I add `build_always_stale: true` option and then the schedule file will be generated every time.
```
$ meson test -C ../pgroonga.build
$ grep regenerated ../pgroonga.build/schedule || echo "(not found)"
(not found
$ touch sql/regenerated.sql expected/regenerated.out
$ meson test -C ../pgroonga.build
$ grep regenerated ../pgroonga.build/schedule || echo "(not found)"
test: regenerated
```